README.md

```diff
@@ -208,4 +208,7 @@ hermes.call("length", [new BABYLON.Vector2(2, 0)]).then(result => {
 ### Changelog
 
 * **v1.0.0:** Initial release
-* **v1.0.1:** Add latest file in npm
\ No newline at end of file
+* **v1.0.1:** Add latest file in npm
+* **v1.0.2:** 
```

Comment:
You changed the version to 1.1.0 in package.json. You should update your changelog

Reply:
Oops it's an omission, i clean this

Reply:
Done